tests/coprocessor/test_select.rs

```diff
@@ -0,0 +1,260 @@
+use tikv::server::coprocessor::*;
+use kvproto::kvrpcpb::Context;
+use tikv::util::codec::{table, Datum, datum, number};
+use tikv::storage::{Dsn, Mutation, Key};
+use tikv::storage::engine::{self, Engine};
+use tikv::storage::txn::TxnStore;
+use kvproto::coprocessor::{Request, KeyRange};
+use tipb::select::{SelectRequest, SelectResponse};
+use tipb::schema::{self, ColumnInfo};
+
+use std::ops::RangeFrom;
+use std::sync::Arc;
+use std::{slice, i64};
+use protobuf::{RepeatedField, Message};
+
+const TYPE_VAR_CHAR: u8 = 1;
+const TYPE_LONG: u8 = 2;
+
+struct TableInfo {
+    t_id: i64,
+    c_types: Vec<u8>,
+    c_ids: Vec<i64>,
+    indices: Vec<i32>,
+    i_ids: Vec<i64>,
+}
+
+impl TableInfo {
+    fn as_pb_table_info(&self) -> schema::TableInfo {
+        let mut tb_info = schema::TableInfo::new();
+        tb_info.set_table_id(self.t_id);
+        let mut c_info = ColumnInfo::new();
+        c_info.set_tp(TYPE_LONG as i32);
```

Comment:
My fault, @coocood  has said that int64 is the value type, seems TypeLong/TypeLongLong are both OK.